### ISSUE TITLE
Form fields missing labels

### ISSUE THREAD

**Issue by briri** [DMPRoadmap/roadmap#1445]
Some of the form fields do not have labels. A user using a screen reader cannot determine what these fields are supposed to contain. The audit only included the researcher and public facing pages so we should also go through the admin pages and make sure that they all have appropriate labels.
- Edit plan page
- Edit profile page
- Download plan page

This feedback was provided from the UCOP electronic accessibility review. It was defined as having 'High' priority.

**Comment by stephaniesimms** [comment]
can't test UI visually. closing issue.
